.github/workflows/pip-packaging-linux.yml

```diff
@@ -0,0 +1,60 @@
+name: pip packaging linux
+
+on:
+  push: {}
+  pull_request: {}
+
+# on:
+#   release:
+#     types: [published]
+
+jobs:
+  build:
+    name: build pip wheels
+    runs-on: ubuntu-latest
+    container: gudhi/pip_for_gudhi
```

Comment:
It is a bit confusing to have ubuntu-latest immediately followed by a docker using some old linux, but if that's how it works...